spec/hermod/xml_section_builder/datetime_node_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "minitest_helper"
+
+module Hermod
+  describe XmlSection do
+
+    DateTimeXml = XmlSection.build(formats: {datetime: "%Y-%m-%d %H:%M:%S"}) do |builder|
```

Comment:
Oh good plan. In fact, that's what happens if I leave that bit out so...

45436e3